js/sdks/packages/log4shell/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
There's no JS in this tool whatsoever, and its not really one of our SDKs.  We can publish packages out of the `/tools` directory (nothing stopping us from doing that).  That would make a bit more sense to me, personally.